packages/workflow-upgrade/index.js

```diff
@@ -33,27 +51,23 @@ module.exports = cwd => {
       delete devDependencies['eslint-plugin-react'];
       delete devDependencies['@availity/workflow-plugin-react'];
       delete devDependencies['@availity/workflow-plugin-angular'];
+
+      // Get needed dependencies from eslint-config-availity
+      Logger.info('Adding peerDependencies from eslint-config-availity to devDependencies');
+      exec(`${installer} info eslint-config-availity peerDependencies`, (error, stdout, stderr) => {
```

Comment:
It seems like this block doesn't work successfully every time, trying to figure out something more consistent. I don't want to just add them like we do with `@availity/workflow` and `eslint-config-availity` in case the versions start to outpace our updates and become incompatible.

Reply:
can this happen only if the app is typescript? looks like from the eslint config the peer deps are only required if its ts

Reply:
maybe wrapping this block in a if(dependencies['typescript']) ?

Reply:
Didn't have a chance to revisit this yet but I know for Node apps the `eslint-config-availity` peer dependencies are optional, but for our React apps that extend `availity/workflow` or `availity/browser` configs, the way it's set up now it doesn't matter whether or not you are using Typescript, the eslint config is still [requiring those parsers ](https://github.com/Availity/eslint-config-availity/blob/master/browser.js#L4)